tests/test_create_time_dimensions.py

```diff
@@ -145,6 +156,12 @@ def test_time_dimension_model4(annual_time_dimension_model):
     check_register_annual_time(annual_time_dimension_model)
 
 
+def test_time_dimension_model5(representative_time_dimension_model):
+    config = RepresentativePeriodTimeDimensionConfig(representative_time_dimension_model)
+    config.list_expected_dataset_timestamps()
+    config.get_time_ranges()
```

Comment:
Is there something we should check about what's returned here? In any case, thanks for adding this one--seems like it was needed!